easybuild/easyconfigs/k/Kent_tools/Kent_tools-337-linux.x86_64.eb

```diff
@@ -0,0 +1,32 @@
+#This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#Author: Ravi Tripathi
+#Email: ravi89@uab.edu
+
+easyblock = "PackedBinary"
+
+name = 'Kent_tools'
+version = '337'
+versionsuffix = '-linux.x86_64'
+
+homepage = 'http://genome.cse.ucsc.edu/'
+description = """Kent tools: collection of tools used by the UCSC genome browser."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# download from http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64/, and pack into tarball with:
+#   mkdir Kent_tools
+#   rsync -aP rsync://hgdownload.cse.ucsc.edu/genome/admin/exe/linux.x86_64/ Kent_tools
+#   tar cfvz Kent_tools-20130806.tar.gz Kent_tools
+source_urls = ['http://hgdownload.soe.ucsc.edu/admin/exe/']
+sources = ['userApps.v%(version)s.src.tgz']
```

Comment:
@rtripath89 ah, sorry, I missed the fact that you're actually building the tools here than just picking up a binary release

In that case, we shouldn't be using the `dummy` toolchain, since we're at the mercy of the OS in that case?

Also, another easyblock than `PackedBinary` would be better, probably `MakeCp`?

I'm happy to rename this to USCS Tools if that's the more common name, since there's no clear 'official' name here...

Thoughts @verdurin?